modules/installation-osp-deploying-bare-metal-preexisting.adoc

```diff
@@ -0,0 +1,73 @@
+[id="installation-osp-deploying-bare-metal-preexisting_{context}"]
+= Configuring a cluster to use bare-metal compute machines
+
+If you want your cluster to use bare-metal compute machines, modify the
+`install-config.yaml` file. In this configuration, control plane machines are
+deployed on VMs that use user-provisioned infrastructure.
```

Comment:
Same comment as above. Only the networking infrastructure is provided by the user in this case.